homeassistant/components/kodi/media_player.py

```diff
@@ -275,20 +271,32 @@ class KodiEntity(MediaPlayerEntity):
         | MediaPlayerEntityFeature.VOLUME_STEP
     )
 
-    def __init__(self, connection, kodi, name, uid):
+    def __init__(self, connman: KodiConnectionManager, name: str, uid: str) -> None:
         """Initialize the Kodi entity."""
-        self._connection = connection
-        self._kodi = kodi
+        super().__init__(connman)
+        self._kodi = connman.kodi
         self._unique_id = uid
         self._players = None
-        self._properties = {}
-        self._item = {}
-        self._app_properties = {}
+        self._properties: dict = {}
+        self._item: dict = {}
+        self._app_properties: dict = {}
         self._media_position_updated_at = None
         self._media_position = None
-        self._connect_error = False
 
-        self._attr_name = name
+        self._conf_name = name
+        self._attr_name = "Kodi"
```

Comment:
Since the media player is the main entity for the device you can simply set the name to `None`, and it would take the device name. You can do it at the class level, because it doesn't change between instances.